sktime/libs/granite_ttm/__init__.py

```diff
@@ -0,0 +1,218 @@
+"""Python implementation TinyTimeMixer Foundation Model.
+
+Unofficial fork of the ``ibm-granite/granite-tsfm`` package, maintained in ``sktime``.
```

Comment:
perhaps add, temporary with a view to moving to a potential package release of ttm

Reply:
Added